### DISCUSSION TITLE
Playbook/Sigma: Search on CIDR ranges

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#7368] [Unsupported Versions] [answered]
Hi all,

How are you searching for ip ranges on playbooks/sigma? 

I.e 10.0.0.0/21

**Comment by defensivedepth** [comment]
hey there!

Can you please give a little more context? Is there something not working the way you would expect?

**Comment by wildlyunder** [comment, marked as answer]
The solution is to do the following
      - ' 10.' #10.0.0.0/8
      - ' 192.168.' #192.168.0.0/16
      -  '192.168.0' #192.168.0.0/24
      - ' 172.16.' #172.16.0.0/12
      - ' 172.17.'
      - ' 172.18.'
      - ' 172.19.'
      - ' 172.20.'
      - ' 172.21.'
      - ' 172.22.'
      - ' 172.23.'
      - ' 172.24.'
      - ' 172.25.'
      - ' 172.26.'
      - ' 172.27.'
      - ' 172.28.'
      - ' 172.29.'
      - ' 172.30.'
      - ' 172.31.'
      - ' 127.' #127.0.0.0/8
      - ' 169.254.' #169.254.0.0/16
  
You can't add CIDR ranges into SIGMA.
